tests/baselines/reference/APISample_watcher.js

```diff
@@ -169,9 +169,7 @@ function watch(rootFileNames, options) {
         });
     }
     function logErrors(fileName) {
-        var allDiagnostics = services.getCompilerOptionsDiagnostics()
-            .concat(services.getSyntacticDiagnostics(fileName))
-            .concat(services.getSemanticDiagnostics(fileName));
+        var allDiagnostics = services.getCompilerOptionsDiagnostics().concat(services.getSyntacticDiagnostics(fileName)).concat(services.getSemanticDiagnostics(fileName));

```

Comment:
This seems like a step backward. It seems like we need to figure out the indentation in `emitter.ts` as if we had parsed the `DotToken`.